python/cudf/cudf/pylibcudf_tests/test_string_capitalize.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+import pyarrow as pa
+import pytest
+from utils import assert_column_eq
+
+import cudf._lib.pylibcudf as plc
+
+
+@pytest.fixture(scope="module")
+def pa_data():
+    data = [
+        "leopard",
+        "Golden Eagle",
+        "SNAKE",
+        "",
+        "!A",
+        "hello World",
+        "A B C",
+        "#",
+        "AƻB",
+        "Ⓑⓖ",
+        "Art of War",
+        "The quick bRoWn fox juMps over the laze DOG",
+        '123nr98nv9rev!$#INF4390v03n1243<>?}{:-"',
+        "accénted",
+        None,
+    ]
+    return pa.array(data)
+
+
+@pytest.fixture(scope="module")
+def plc_data(pa_data):
+    return plc.interop.from_arrow(pa_data)
+
+
+def test_capitalize(plc_data, pa_data):
+    got = plc.strings.capitalize.capitalize(plc_data)
+    expected = pa.compute.utf8_capitalize(pa_data)
+    assert_column_eq(got, expected)
+
+
+def test_title(plc_data, pa_data):
+    # A sequence shall be all characters that are not a space
+    # matches arrow for now
+    str_char_type = (
+        plc.strings.char_types.StringCharacterTypes.ALL_TYPES
+        & ~plc.strings.char_types.StringCharacterTypes.SPACE
+    )
```

Comment:
cc @vyasr for our offline conversation about this failing test. I would have thought the latest commit that plumbed this enum would have fixed this test, but currently passing this enum to the pylibcudf API seems to have no effect, possibly due to a bug somewhere.

Reply:
This has been resolved.